src/Products/Annotation/Annotator/TextRedactionAnnotator.cs

```diff
@@ -1,39 +1,55 @@
-﻿using GroupDocs.Annotation.Domain;
+﻿using GroupDocs.Annotation.Models;
+using GroupDocs.Annotation.Models.AnnotationModels;
 using GroupDocs.Annotation.MVC.Products.Annotation.Entity.Web;
+using GroupDocs.Annotation.Options;
 using System;
+using System.Collections.Generic;
 
 namespace GroupDocs.Annotation.MVC.Products.Annotation.Annotator
 {
-    public class TextRedactionAnnotator : TextAnnotator
+    public class TextRedactionAnnotator : TextHighlightAnnotation
     {
-        public TextRedactionAnnotator(AnnotationDataEntity annotationData, PageData pageData)
-            : base(annotationData, pageData)
+        private TextRedactionAnnotation textRedactionAnnotation;
+
+        public TextRedactionAnnotator(AnnotationDataEntity annotationData, PageInfo pageInfo)
+            : base(annotationData, pageInfo)
         {
+            this.textRedactionAnnotation = new TextRedactionAnnotation()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove these redundant parentheses.](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372471)